Source/Core/AudioCommon/Mixer.cpp

```diff
@@ -1,9 +1,11 @@
 // Copyright 2008 Dolphin Emulator Project
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
+// Modified For Ishiiruka By Tino
 
 #include "AudioCommon/AudioCommon.h"
 #include "AudioCommon/Mixer.h"
+#include "Common/Atomic.h"
```

Comment:
Unless I misread/missed anything, this can be replaced with an `<atomic>` include